app/models/serializable_facility.rb

```diff
@@ -0,0 +1,17 @@
+class SerializableFacility < SimpleDelegator
```

Comment:
This seemed like a fairly clean way of making the cross-facility (i.e. Facility with a nil ID) serializable by ActiveJob. I'm definitely open to other ways. I considered just passing IDs, but that would involve extra logic in the Job deciding how to look it up anyways.